completion/available/composer.completion.bash

```diff
@@ -1,133 +1,129 @@
-#!/usr/bin/env bash
+# shellcheck shell=bash
+cite "about-completion"
+about-completion "composer completion"
 
-_composer()
-{
-    local cur script coms opts com
-    COMPREPLY=()
-    _get_comp_words_by_ref -n : cur words
+function _composer() {
```

Comment:
Could you change this to `__composer_complete` or `__composer_completion`?

Reply:
done